x-pack/plugin/src/test/resources/rest-api-spec/test/enrich/10_basic.yml

```diff
@@ -0,0 +1,15 @@
+---
+"Test enrich crud apis":
+
+  - do:
+      enrich.put_policy:
+        name: policy-crud
+        body:
+          policy:
```

Comment:
The policy field isn't needed and this is why this test is failing in pr ci.

Reply:
oh geez, i removed that `{"policy": ` wrapper I had initially put around the PUT... thank you for the catch.